DevelopGitHub/GitHubExtension/GitHubExtention.Preferences.WebApi/Azure/AzureBlobStorageMultipartProvider .cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.IO;
+using System.Net.Http;
+using System.Threading.Tasks;
+using GitHubExtention.Preferences.WebApi.Queries;
+using Microsoft.WindowsAzure.Storage.Blob;
+
+namespace GitHubExtention.Preferences.WebApi
+{
+    public class AzureBlobStorageMultipartProvider : MultipartFileStreamProvider
+    {
+        private IAzureContainerQuery _container;
+        private string _fileLocation;
+
+        public AzureBlobStorageMultipartProvider(IAzureContainerQuery container)
```

Comment:
Nice injection...Why you are not using the same approach with MultipartProvider?